src/app/zap-templates/templates/app/cluster-enums.zapt

```diff
@@ -24,7 +24,7 @@ enum class {{asType label}} : {{asUnderlyingZclType name}} {
 {{#zcl_enum_items}}
 k{{asUpperCamelCase label}} = {{asHex value 2}},
 {{/zcl_enum_items}}
-// kUnknownEnumValue = {{first_unused_enum_value mode="first_unused"}},
```

Comment:
Oh, it was already there...

Reply:
Well, as of very recently, and commented out because it failed to compile without some of the other changes in this PR.